README.md

```diff
@@ -18,50 +18,71 @@ to make public data accessible to everyone.
 
 To see the extent of data we have today, [browse the Knowledge Graph](https://datacommons.org/browser).
 
-## Developing locally
+## Markdown
+
+The Data Commons documentation uses [Kramdown](https://kramdown.gettalong.org/syntax.html) Markdown.
+
+## Build locally
 
 The documentation site is built using Jekyll. To run this locally:
 
 1. Install [Ruby](https://jekyllrb.com/docs/installation/)
 1. Run `bundle update`
 1. Run `bundle exec jekyll serve --incremental`
 
-If you make changes to yml files, re-run `bundle exec jekyll serve`
+You can continue to make local changes and just refresh the browser. If you make changes to YAML files, re-run `bundle exec jekyll serve`
+
+Tip: If you want to make the staged site accessible to others (and not just on the loopback), add `--host 0.0.0.0` to the command. Then, users can access the site using the hostname of the machine where the site is running.
 
 ## License
 
 Apache 2.0
 
-## Contributing
+## Contribute changes
+
+### One-time setup steps
+
+1. In https://github.com/datacommonsorg/docsite, click the **Fork** button to fork the repo.
+1. Clone your forked repo to your desktop:
 
-In https://github.com/datacommonsorg/docsite, click on "Fork" button to fork the repo.
+    <pre>
+    git clone https://github.com/<var>USER_NAME</var>/docsite
+    </pre>
 
-Clone your forked repo to your desktop.
+    This adds your fork as the remote called `origin`.
 
-Add datacommonsorg/docsite repo as a remote:
+1. Add `datacommonsorg/docsite` repo as a remote:
 
-```shell
-git remote add dc https://github.com/datacommonsorg/docsite.git
-```
+    <pre>
```

Comment:
Any reason why the backticks don't work as well here? Is it to get the var formatting?

Reply:
Yep. Google Dev style guide requires italics for variables. Markdown uses _ which gets rendered literally if you use the backticks. It's kind of a pain, but that's what we have.